src/tests/conformance/cmd/squelch.rs

```diff
@@ -101,15 +195,36 @@ async fn c009_TM_SQUELCH_cannot_squelch_peer_ledger_proposals() {
                 },
             ) = synth_node.recv_message().await
             {
-                if validator_pub_key == node_pub_key {
-                    break;
+                if distant_node_keys.contains(&node_pub_key) {
+                    panic!("It shouldn't be possible to receive proposing legders from squelched nodes.");
                 }
             }
         }
     })
-    .await
-    .expect("TmProposeLedger not received in time");
+    .await.is_err());
```

Comment:
Hmm, is there any better way to do this?

Reply:
the `async` block could be bound to a variable; it might make things more visually nice; however, this check might not suffice taking https://github.com/runziggurat/xrpl/pull/57/files#r990972790 into account